### PR TITLE
Checking for integer instead of int for summable columns.

### PR DESCRIPTION
**Pull request by manwithtwowatches**

### CODE REVIEW

**Comment by myabc** [comment]
@manwithtwowatches added specs (second commit #0bfd783 actually tests the change you made). If they looks good to you, I'll merge.

File: spec/controllers/api/v3/work_packages_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      it 'assigns column metadata' do
+        get :column_data, format: 'xml', ids: [1, 2],
+          column_names: %w(subject status estimated_hours story_points)
+
+        expect(assigns(:columns_meta)).to have_key('group_sums')
+        expect(assigns(:columns_meta)).to have_key('total_sums')
+
+        expect(assigns(:columns_meta)['total_sums'].size).to eq(4)
+        expect(assigns(:columns_meta)['total_sums'][2]).to eq(10.0)
+        expect(assigns(:columns_meta)['total_sums'][3]).to eq(3)
```
Comment (myabc) [inline spec/controllers/api/v3/work_packages_controller_spec.rb]: n.b. This doesn't exactly adhere to the "one assertion per example" principle. Examples should be reorganised at a later point in time.

File: spec/controllers/api/v3/work_packages_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        allow(Setting).to receive(:work_package_list_summable_columns).and_return(
+          %w(estimated_hours story_points)
+        )
+        WorkPackage.stub_chain(:visible, :find) {
+          FactoryGirl.create_list(:work_package, 2, estimated_hours: 5, story_points: 3)
+        }
+      end
+
+      xit 'handles incorrect column names' do
+        get :column_data, format: 'xml', ids: [1, 2], column_names: %w(non_existent status)
```
Comment (myabc) [inline spec/controllers/api/v3/work_packages_controller_spec.rb]: n.b. this should be fixed. Passing incorrect `column_names` will currently result in the client getting an 500 Error.

```
     NoMethodError:
       undefined method `non_existent' for #<WorkPackage:0x007fb48191aec8>
```

File: spec/controllers/api/v3/work_packages_controller_spec.rb (hunk 1 above)
Comment (manwithtwowatches) [inline spec/controllers/api/v3/work_packages_controller_spec.rb]: @myabc this one fails for me. total sum for story points should be 6 no?

Comment (myabc) [inline spec/controllers/api/v3/work_packages_controller_spec.rb]: @manwithtwowatches yep, good catch. It should have been 6.

**Pull request merged**